src/libraries/System.IO.Pipelines/Directory.Build.props

```diff
@@ -2,6 +2,5 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <StrongNameKeyId>Open</StrongNameKeyId>
-    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
Same for this one: https://github.com/dotnet/runtime/blob/master/src/libraries/NetCoreAppLibrary.props#L4